plugins/greatexpectations/flytekitplugins/greatexpectations/task.py

```diff
@@ -0,0 +1,166 @@
+        self._expectation_suite = expectation_suite
+        self._batch_request = task_config
+        self._data_context = data_context
+        self._checkpoint_params = checkpoint_params
+
+        super(GETask, self).__init__(
+            name=name,
+            task_config=task_config,
+            task_type=_TASK_TYPE,
+            interface=Interface(inputs=inputs),
+            **kwargs,
+        )
+
+    def execute(self, **kwargs) -> typing.Any:
+        context = ge.data_context.DataContext(self._data_context)
+
+        if len(self.python_interface.inputs.keys()) != 1:
+            raise RuntimeError(f"Expected one input argument to validate the dataset")
+
+        try:
+            dataset = kwargs[list(self.python_interface.inputs.keys())[0]]
+        except KeyError:
+            raise KeyError(f"Your kwargs key has to be {list(self.python_interface.inputs.keys())[0]}")
+
+        if type(dataset) != str:
+            raise RuntimeError(f"'dataset' has to have string data type")
+
+        final_batch_request = {
+            "data_asset_name": dataset,
+            "datasource_name": self._data_source,
+            "data_connector_name": self._data_connector,
+        }
+
+        # GE's RuntimeBatchRequest
+        if self._batch_request and self._batch_request.runtime_parameters:
+            final_batch_request.update(
+                {
+                    "runtime_parameters": self._batch_request.runtime_parameters,
+                    "batch_identifiers": self._batch_request.batch_identifiers,
+                    "batch_spec_passthrough": self._batch_request.batch_spec_passthrough,
+                }
+            )
+
+        # GE's BatchRequest
+        elif self._batch_request:
+            final_batch_request.update(
+                {
+                    "data_connector_query": self._batch_request.data_connector_query,
+                }
+            )
+
+        checkpoint_config = {
+            "class_name": "SimpleCheckpoint",
+            "validations": [
+                {
+                    "batch_request": final_batch_request,
+                    "expectation_suite_name": self._expectation_suite,
+                }
+            ],
+        }
+
+        if self._checkpoint_params:
+            checkpoint = SimpleCheckpoint(
+                f"_tmp_checkpoint_{self._expectation_suite}", context, **checkpoint_config, **self._checkpoint_params
+            )
+        else:
+            checkpoint = SimpleCheckpoint(f"_tmp_checkpoint_{self._expectation_suite}", context, **checkpoint_config)
+
+        # identify every run uniquely
+        run_id = RunIdentifier(
+            **{
+                "run_name": self._data_source + "_run",
+                "run_time": datetime.datetime.utcnow(),
+            }
+        )
+
+        checkpoint_result = checkpoint.run(run_id=run_id)
+        final_result = convert_to_json_serializable(checkpoint_result.list_validation_results())[0]
```

Comment:
can we not send this as an output of the task?

Reply:
We can send it.